test/hotspot/jtreg/ProblemList-Xcomp.txt

```diff
@@ -38,8 +37,6 @@ serviceability/AsyncGetCallTrace/MyPackage/ASGCTBaseTest.java 8303168 linux-x64
 
 serviceability/sa/TestJhsdbJstackMixed.java 8248675 linux-aarch64
 
-serviceability/jvmti/VMObjectAlloc/VMObjectAllocTest.java 8288430 generic-all
```

Comment:
If the 8288430 is a dup of 8277573 then should we close it as such?

Reply:
Done.